Show wrangler error messages in job step

If there is something wrong with wrangler configuration, for examples bindings defined in root missing in environment used in action, you will get this non-informative and misleading error:

```
Run cloudflare/wrangler-action@v3
🔍 Checking for existing Wrangler installation

🚀 Running Wrangler Commands
Error: The process '/home/runner/setup-pnpm/node_modules/.bin/pnpm' failed with exit code 1
Error: 🚨 Action failed
```

Real issue is can be found in raw logs, but as message shown is completely unrelated to error it takes a lot of time to figure it out.

I think it will be good to show issues that happen in wrangler inside wrangler action step.

I'm dealing with the same problem with this workflow config:
```
name: Migrate preview database

on:
  workflow_dispatch:
  pull_request:
    branches: [main]
    paths:
      - 'migrations/**'
  push:
    branches: [preview]
    paths:
      - 'migrations/**'

jobs:
  migrate_preview_database:
    name: Migrate preview database
    runs-on: ubuntu-latest

    steps:
      - uses: actions/checkout@v4
      - uses: pnpm/action-setup@v4
      - uses: actions/setup-node@v4
        with:
          node-version: 22
          cache: 'pnpm'

      - name: 🕊️ Migrate 
        uses: cloudflare/wrangler-action@v3
        with:
          apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
          accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
          command: d1 migrations apply mydb --remote --env preview
          wranglerVersion: "4.15.2"
```

Where are those raw logs located @brachkow? When I click here

![Image](https://github.com/user-attachments/assets/47debbfb-f82f-441a-a87e-f7fe5b5f7cec)

I see no useful error information
```
2025-06-18T14:36:32.3389781Z ✅ Wrangler installed
2025-06-18T14:36:32.3391822Z ##[endgroup]
2025-06-18T14:36:32.3397500Z ##[group]🚀 Running Wrangler Commands
2025-06-18T14:36:32.3425063Z [command]/home/runner/setup-pnpm/node_modules/.bin/pnpm exec wrangler d1 migrations apply alma-preview --remote --env preview
2025-06-18T14:36:33.5581992Z 
2025-06-18T14:36:33.5585255Z  ⛅️ wrangler 4.15.2 (update available 4.20.2)
2025-06-18T14:36:33.5586314Z ---------------------------------------------
2025-06-18T14:36:33.5587095Z 
2025-06-18T14:36:33.9800633Z ##[endgroup]
2025-06-18T14:36:33.9832331Z ##[error]The process '/home/runner/setup-pnpm/node_modules/.bin/pnpm' failed with exit code 1
2025-06-18T14:36:33.9843725Z ##[error]🚨 Action failed
2025-06-18T14:36:33.9915857Z Post job cleanup.
```

--- a/.changeset/surface-wrangler-errors.md
+++ b/.changeset/surface-wrangler-errors.md
@@ -1,0 +1,5 @@
+---
+"wrangler-action": patch
+---
+
+Surface wrangler error messages in the action step output instead of showing only a generic failure message.

--- a/src/wranglerAction.ts
+++ b/src/wranglerAction.ts
@@ -354,7 +354,18 @@ async function wranglerCommands(
 			};
 
 			// Execute the wrangler command
-			await exec(`${packageManager.exec} wrangler ${command}`, args, options);
+			try {
+				await exec(
+					`${packageManager.exec} wrangler ${command}`,
+					args,
+					options,
+				);
+			} catch (err: unknown) {
+				if (stdErr) {
+					error(config, stdErr);
+				}
+				throw err;
+			}
 
 			// Set the outputs for the command
 			setOutput("command-output", stdOut);

--- a/src/wranglerAction.ts
+++ b/src/wranglerAction.ts
@@ -393,7 +393,18 @@ async function wranglerCommands(
 			};
 
 			// Execute the wrangler command
-			await exec(`${packageManager.exec} wrangler ${command}`, args, options);
+			try {
+				await exec(
+					`${packageManager.exec} wrangler ${command}`,
+					args,
+					options,
+				);
+			} catch (err: unknown) {
+				if (stdErr) {
+					error(config, stdErr);
+				}
+				throw err;
+			}
 
 			// Set the outputs for the command
 			setOutput("command-output", stdOut);

